Mustache/src/html/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<head>
+    <meta charset="utf-8">
+    <title>Agentes</title>
+    <link rel="stylesheet" href="main.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script type="text/javascript" src="../js/main.js"></script>

```

Comment:
Load this scripts at the bottom of the body section. 
https://faqs.skillcrush.com/article/176-where-should-js-script-tags-be-linked-in-html-documents